composer.json

```diff
@@ -80,13 +80,16 @@
     ],
     "post-update-cmd": [
       "Contributte\\Neonizer\\NeonizerExtension::process"
+    ],
+    "set-config": [
```

Comment:
Proc tahle zmena?

Reply:
jak bys to udělal, když je jiný host v doclkeru a v GA?

Reply:
aha, ono to funguje i bez toho

Reply:
to nepůjde přes sqlite3, není v tom PHP imagi

Reply:
Ikdyž jsem ho tam přidal, nejede to, musely by vytvořit migrace pro sqlit3.  Já bych celé sqlite zahodil.

Reply:
Vubec to tam nedavej tohle.